14-EventToCommandBehavior/src/PrismSample/Views/EventArgsParameterExamplePage.xaml

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="PrismSample.Views.EventArgsParameterExamplePage"
+             Title="Event Args Parameter Example"
+             
+             xmlns:b="http://prismlibrary.com">
+
+  <ListView ItemsSource="{Binding Developers}">
+    <ListView.ItemTemplate>
+      <DataTemplate>
+
+        <TextCell Text="{Binding FullName}"
+                    Detail="{Binding Country}" />
+
+      </DataTemplate>
+    </ListView.ItemTemplate>
+
+    <ListView.Behaviors>
+      <b:EventToCommandBehavior EventName="ItemTapped" 
+                                Command="{Binding SelectedDeveloperCommand}"
+                                EventArgsParameterPath="Item" />
+    </ListView.Behaviors>
+
+  </ListView>
+
+</ContentPage>
```

Comment:
see comments for EventArgsConverterExamplePage

Reply:
Done with the latest commit. :)